src/dbErrorHandlers.js

```diff
@@ -0,0 +1,45 @@
+const dbErrorHandlers = module.exports = {}
+
+dbErrorHandlers.custom = (err, req, res, next) => {
+  if (err.name !== 'CustomDbError') {
+    return next(err)
+  }
+  switch (err.message) {
+    case 'No document matching that id':
```

Comment:
Okay, could you move all these stings into constants

ie
```js
// src/constants/error_messages.js

module.exports = {
  errorNames: {
    CUSTOM: 'CustomDbError',
    MONGO: 'MongoError'
  },
  messages: {
    ID_NOT_FOUND_CUSTOM: 'No document matching that id'
    ...
  }
}
```
Why?
  1. This is fragile, and prone to typos
  2. If you want to change the text, you are likely to forget somewhere